sklearn/covariance/_graph_lasso.py

```diff
@@ -188,7 +188,7 @@ def graphical_lasso(
     GraphicalLassoCV : Sparse inverse covariance with
         cross-validated choice of the l1 penalty.
```

Comment:
You need to modify these 2 lines as reported in the issue.

Reply:
Uhm actually no, this one is correct, it is in the class `GraphicalLassoCV`, the section `See Also` that you need to make the change.